.github/workflows/ci.yml

```diff
@@ -123,6 +123,9 @@ jobs:
         wget https://repo.msys2.org/mingw/mingw64/mingw-w64-x86_64-gcc-13.2.0-6-any.pkg.tar.zst https://repo.msys2.org/mingw/mingw64/mingw-w64-x86_64-gcc-libs-13.2.0-6-any.pkg.tar.zst
         pacman -U --noconfirm mingw-w64-x86_64-gcc-13.2.0-6-any.pkg.tar.zst mingw-w64-x86_64-gcc-libs-13.2.0-6-any.pkg.tar.zst
 
+    - name: Install dependencies (All)
+      run: cargo install lychee
```

Comment:
I'm leaning towards not mentioning this in `CONTRIBUTING.md` since:
- Wrong links are infrequent
- This CI will show them anyway